weather-app/main.js

```diff
@@ -0,0 +1,13 @@
+const geodata = require('./helpers/geodata')
+const weather = require('./helpers/getWeather')
+
+geodata.getGeoData("Belarus", (err, data) => {
```

Comment:
Try to pass empty string instead of 'Belarus' and see what happens, also if you started using '' use everywhere '' not ""